src/binding/python/openPMD.cpp

```diff
@@ -55,11 +57,12 @@ PYBIND11_MODULE(openPMD, m) {
         openPMDapi << "-" << OPENPMDAPI_VERSION_LABEL;
     m.attr("__version__") = openPMDapi.str();
 
-    // variants
-    m.attr("have_mpi") = bool(openPMD_HAVE_MPI);
-    m.attr("have_hdf5") = bool(openPMD_HAVE_HDF5);
-    m.attr("have_adios1") = bool(openPMD_HAVE_ADIOS1);
-    m.attr("have_adios2") = bool(openPMD_HAVE_ADIOS2);
-    // m.attr("have_json") = bool(openPMD_HAVE_JSON);
+    // feature variants
+    m.attr("feature") = std::map<std::string, bool>{
```

Comment:
@C0nsultant last question: plural *features*? :)

Reply:
To be consistent with the other containers in the API, plural is be preferred.